design-docs/0002-uds-ui.md

```diff
@@ -0,0 +1,30 @@
+# UDS UI
+
+Author(s): @decleaver  
+Date Created: Sept 9, 2024  
+Status: IMPLEMENTED  
+Ticket: https://github.com/defenseunicorns/uds-cli/issues/870  
+
+### Problem Statement
+
+The goal of the `uds ui` command is to allow uds-cli users to launch the UDS Runtime application from the command line.
+
+### Proposal
+
+Bundle the UDS Runtime binaries as a part of uds-cli allowing users to launch UDS Runtime from the command line.
+
+### Scope and Requirements
+
+Allow users to launch UDS Runtime from uds-cli
+
+### Implementation Details
+
+To execute UDS Runtime from the `uds-cli`, the appropriate runtime binary is pulled into the `src/cmd/bin` directory during the build process. This is based on the specific build task being run (e.g., `build-cli-linux-arm`). The runtime binary is then packaged with the `uds-cli` binary.
+
+To ensure the `uds ui` command works correctly, the `uds-cli` must be built using the build tasks specified in `tasks.yaml`. If not, the runtime binary might be missing, causing the `uds ui` command to fail.
+
+### Alternatives Considered
```

Comment:
I'd like to include another considered alternative detailing why vendoring doesn't work in this situation. Happy to write it myself and push to this branch

Reply:
I can add some words, but i didn't try going down that route, so feel free to additional info on what you found.